packages/twenty-front/src/modules/workflow/workflow-steps/workflow-actions/email-action/extensions/resizable-image/ResizableImageView.tsx

```diff
@@ -0,0 +1,207 @@
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { type NodeViewProps, NodeViewWrapper } from '@tiptap/react';
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+import { isDefined } from 'twenty-shared/utils';
+
+const IMAGE_MIN_WIDTH = 32;
+const IMAGE_MAX_WIDTH = 600;
+
+const StyledNodeViewWrapper = styled(NodeViewWrapper)`
+  height: 100%;
+  ${({ align }) => {
+    switch (align) {
+      case 'left':
+        return css`
+          margin-left: 0;
+        `;
+      case 'right':
+        return css`
+          margin-right: 0;
+        `;
+      case 'center':
+        return css`
+          margin-left: auto;
+          margin-right: auto;
+        `;
+    }
+  }}
+`;
+
+const StyledImageWrapper = styled.div<{ width?: number }>`
+  height: 100%;
+`;
+
+const StyledImageContainer = styled.div`
+  max-width: 100%;
+  position: relative;
+`;
+
+const StyledImage = styled.img`
+  height: auto;
+  max-width: 100%;
+`;
+
+const StyledImageHandle = styled.div<{ handle: 'left' | 'right' }>`
+  border-radius: ${({ theme }) => theme.border.radius.md};
+  background-color: ${({ theme }) => theme.background.primaryInverted};
+  border: 1px solid ${({ theme }) => theme.background.primary};
+  cursor: col-resize;
+  height: ${({ theme }) => theme.spacing(8)};
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+  width: ${({ theme }) => theme.spacing(2)};
```

Comment:
Feels odd to use spacing for a width? Not sure what it matches exactly but I feel there is probably better